src/services/ApolloService/ApolloService.js

```diff
@@ -0,0 +1,5 @@
+import ApolloClient from 'apollo-boost';
```

Comment:
Please, rename `AppolloService` to just `client.js` and put it in some `util` folder. It is not a folder

Reply:
Renamed and moved to utils folder